mods/d2k/sequences/infantry.yaml

```diff
@@ -196,10 +196,15 @@ medic: # actually thumper
 		Facings: -8
 		Transpose: true
 		Tick: 120
-	heal: DATA.R8
+	thump: DATA.R8
 		Start: 1458
 		Length: 5
 		Tick: 480
```

Comment:
Ah, my bad. The `@DEFAULT` added to `WithInfantryBody` meant it didn't override the `WithInfantryBody` inherited from the infantry default.